pkgs/by-name/th/thc-secure-delete/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, gccStdenv # x86_64-darwin and aarch64-darwin cant find C compiler
```

Comment:
Just use the normal `stdenv` and add `makeFlags` as mentioned. Using GCC on Darwin is highly unexpected.

Reply:
Ok